Meta/Lagom/CMakeLists.txt

```diff
@@ -539,6 +539,10 @@ if (BUILD_LAGOM)
         set_target_properties(gml-format_lagom PROPERTIES OUTPUT_NAME gml-format)
         target_link_libraries(gml-format_lagom LagomCore LagomGML LagomMain)
 
+        add_executable(headless_browser_lagom ../../Userland/Utilities/headless-browser.cpp)
+        set_target_properties(headless_browser_lagom PROPERTIES OUTPUT_NAME headless-browser)
+        target_link_libraries(headless_browser_lagom LagomWeb LagomWebSocket LagomHTTP LagomJS LagomGfx LagomMain Threads::Threads)
```

Comment:
`Threads::Threads`?

Reply:
Yeah, it was a copy-paste from the one below but doesn't seem needed for the headless browser, so I removed it. Thanks!